twill-yarn/src/main/java/org/apache/twill/yarn/YarnTwillPreparer.java

```diff
@@ -298,6 +299,29 @@ public TwillPreparer addSecureStore(SecureStore secureStore) {
   public TwillPreparer setLogLevel(LogEntry.Level logLevel) {
     Preconditions.checkNotNull(logLevel);
     this.logLevel = logLevel;
+    Map<String, LogEntry.Level> logLevels = new HashMap<>();
+    logLevels.put(Logger.ROOT_LOGGER_NAME, logLevel);
+    saveLogLevels(Constants.SystemMessages.LOG_ALL_RUNNABLES, logLevels);
```

Comment:
Why use the `LOG_ALL_RUNNABLES` as a runnable name? We have the `twillSpec` in this class, so it's easy to just expand the log level settings for each runnable.